sdk/communication/Azure.Communication.PhoneNumbers/src/Models/CommunicationErrorResponse.cs

```diff
@@ -13,7 +13,6 @@ namespace Azure.Communication.PhoneNumbers
 {
     [CodeGenModel("CommunicationErrorResponse")]
     [CodeGenSuppress("CommunicationErrorResponse", typeof(CommunicationError))]
-    [CodeGenSuppress("Error", typeof(CommunicationError))]
```

Comment:
We cannot modify this file, it would break rest of PhoneNumbers SDK. What we should do is to copy these two types CommunicationError and CommunicationErrorResponse WITH the CodeGenSuppress attributes to our /SipRouting folder, so that autores doesn't generate any methods or properties within these files.

Reply:
So the errors you are hitting are related to the fact that the phoneNumbers project has defined nested swagger files. AFAIK this isn't supported in our tooling and the generated code can become easily out of date as our CI checks will only generate the outer swagger. I suspect the thing that is causing this to break now (when it apparently was working before) is the addition of "x-ms-error-response": true in the swagger. Notably, this is only used for 

/sip:testRoutesWithNumber

in this swagger and should not be necessary. I would suggest testing out the removal of this property and seeing if that fixes things.

Reply:
If this doesn't fix things, the alternative would be to suppress the generation of CommunicationError and CommunicationErrorResponse and replace these with fully custom models. In order to suppress the types, you would need to define a Properties/AssemblyInfo.cs file and use the CodeGenSuppressType attribute, e.g. https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/ai/Azure.AI.Projects/src/Properties/AssemblyInfo.cs